drivers/pinmux/pinmux_npcx.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <drivers/pinmux.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(pimux_npcx, LOG_LEVEL_ERR);
+
+/* Driver config */
+struct npcx_pinctrl_config {
+	/* scfg controller base address */
+	scfg_inst_t base;
+};
+
+/* Default io list which default functionality are not IOs */
+#define DT_DRV_COMPAT nuvoton_npcx_pinctrl_def
+static const struct npcx_alt def_alts[] = DT_NPCX_ALT_ITEMS_LIST(0);
+
+static const struct npcx_pinctrl_config npcx_pinctrl_cfg = {
+	.base = (scfg_inst_t)DT_REG_ADDR(DT_NODELABEL(scfg)),
+};
+
+/* Driver convenience defines */
+#define DRV_CONFIG(dev) \
+	((const struct npcx_pinctrl_config *)(dev)->config_info)
+
+#define HAL_INSTANCE(dev) \
+	(scfg_inst_t)(DRV_CONFIG(dev)->base)
+
+/******************************************************************************/
+/* Pin-control local functions */
+static void npcx_pinctrl_alt_sel(const struct npcx_alt *alt, int alt_func)
+{
+	scfg_inst_t inst = npcx_pinctrl_cfg.base;
+	uint8_t alt_mask = 1 << alt->bit;
+
+	/*
+	 * alt_func = 0 -> GPIO functionality
+	 * alt->inverted -> Set DEVALT bit for GPIO
+	 */
+	if (alt_func ^ alt->inverted)
```

Comment:
I didn't understand if XOR was really needed here. Is this a bitwise XOR or just a boolean != ? If this is a boolean operation can you use !!alt_fun != !!alt->inverted